src/common/components/ui/formik/validationSchemas.ts

```diff
@@ -45,38 +47,38 @@ export const nameValidationSchema = (
   Yup.string()
     .required('Required')
     .matches(CLUSTER_NAME_CHARS_REGEX, {
-      message: CLUSTER_NAME_VALIDATION_MESSAGES.INVALID_VALUE,
+      message: clusterNameValidationMessages(t).INVALID_VALUE,
```

Comment:
Please define a constant at the beginning to avoid calling the function so many times.